app/controllers/application_controller.rb

```diff
@@ -45,4 +45,9 @@ def after_sign_in_path_for(_resource)
       user_root_path
     end
   end
+
+  def destroy_state_states(jwt_id)
```

Comment:
Should be `destroy_stale_states`, but otherwise looks good

Reply:
Too many similar words! 😁